src/server-monitor.ts

```diff
@@ -3,7 +3,7 @@
 import { Client } from './client';
 import { NamedAgent } from './types/named-agent';
 
-const DEFAULT_STANZA_TIMEOUT = 70 * 1000;
+const DEFAULT_STANZA_TIMEOUT = 35 * 1000;
```

Comment:
Hawk's timer was halved from 30 seconds to 15 so I've halved our timeout as well to be conservative and align.